examples/musicPlayer/src/1_schema.ts

```diff
@@ -0,0 +1,60 @@
+import {
+    CoMap,
+    CoList,
+    BinaryCoStream,
+    co,
+    Profile,
+    Account,
+  } from "jazz-tools";
+  
+  export class MusicTrack extends CoMap {
+    file = co.ref(BinaryCoStream);
+    title = co.string;
+    duration = co.number;
+    waveform = co.json<number[]>();
```

Comment:
waveform is a list of numbers that won't never change.

Not sure if `co.json` is the better primitive for this kind of data.

Reply:
if that's the case, that's indeed the best way to do it - just how huge is it? Is it worth putting it in a referenced sub-coMap so we can choose to only load it when needed?

Reply:
> Is it worth putting it in a referenced sub-coMap so we can choose to only load it when needed?

Awesome idea!
Would be great way to show off some load optimizations 🚀